src/engine/SCons/Tool/msvc.xml

```diff
@@ -475,4 +475,38 @@ Valid values are '1' or '0'
 </summary>
 </cvar>
 
+<cvar name="VSWHERE">
+<summary>
+<para>
+Specify the location of vswhere.exe. 
+</para>
+
+<para>
+<literal>vswhere.exe</literal> is used to locate MSVC/MSVS installations.
+It was initially distributed with MSVS 2017. But can detect MSVC 2010 and newer with <literal>-legacy</literal> argument. 
+If &cv-VSWHERE; is set, SCons will use that location.
+</para>
+<para>
+Otherwise SCons will look in the following locations and set VSWHERE to the PATH <literal>vswhere.exe</literal>
+is located.
+</para>
+<para>
+Note that &cv-VSWHERE; must be set in when tool msvc is initialized. This is either in the call to 
+<literal>Environment(VSWHERE='c:/my/path/to/vswhere')</literal> or 
+if Environment created specifying not to automatically initialize any tools or tools which doesn't include msvc or MSVS
```

Comment:
I can't quite make sense of this wording?  If created with an empty tool list, then the VSWHERE processing doesn't happen either, so how is this applicable?